framework/generated/dx12_generators/dx12_wrapper_body_generator.py

```diff
@@ -758,6 +758,11 @@ def write_class_member_def(self, class_info, indent=''):
                 expr += unwrapped_args
             expr += ');\n'
 
+            method_names = ["CreateSwapChainForHwnd", "CreateSwapChainForCoreWindow"]
```

Comment:
Instead of modifying the generators, please add these functions to `capture_overrides.json` and call `UpdateSwapChainSize` from the new override methods in `D3D12CaptureManager`.

`CreateSwapChainForComposition` should also be included.